### PR TITLE
Adding solutions to exercises in 03-create

### PR DESCRIPTION
**Pull request by edenisen**
Adding solutions to exercises in 03-create 'Working With Files and Directories': 'Organizing Directories and Files' and 'Copy with Multiple Filenames' as a part of the Checkout Procedure for the SWC Instructor Training.

### CODE REVIEW

File: _episodes/03-create.md [hunk 1]
```diff
@@ -554,6 +554,15 @@ but it does find the copy in `thesis` that we didn't delete.
 > fructose.dat    sucrose.dat
 > ~~~
 > {: .output}
+>
+> > ## Solution
+> > Jamie needs to move her files `fructose.dat` and `sucrose.dat` to `analyzed` directory:
```
Comment (shwina) [inline _episodes/03-create.md]: to `analyzed` directory > to **the** `analyzed` directory

**Comment by shwina** [comment]
@edenisen Thanks so much for this. Could you please make this small correction?

**Comment by edenisen** [comment]
Thank you @shwina, it's done

**Comment by shwina** [comment]
Thanks!

**Pull request merged**